vm-setup/v1aX_integration_test.yml

```diff
@@ -1,10 +1,10 @@
 ---
-- name: v1alpha2 integration test
+- name: "{{ VERSION }} integration test"
```

Comment:
you actually have CAPI_VERSION from the vars file, read from the env vars, you would not need this new variable

Reply:
done